docs/docusaurus/docs/components/_data.jsx

```diff
@@ -1,5 +1,5 @@
 export default {
   release_version: 'great_expectations, version 0.17.5',
   min_python: '3.7',
```

Comment:
```suggestion
  min_python: '3.8',
```

Reply:
@Kilo59 the python version content was updated in a previous PR. Here is how the `_data.jsx` file currently appears:

![image](https://github.com/great-expectations/great_expectations/assets/104205257/b8433012-3e9a-49db-9fa0-b1763aedb903)

The version information is correct in the [latest version of the documentation](https://docs.greatexpectations.io/docs/tutorials/quickstart/):

![image](https://github.com/great-expectations/great_expectations/assets/104205257/8469a5fa-fc53-44b1-81b3-bd9159703b4c)